ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V113__enable_string_interning.sql

```diff
@@ -0,0 +1,64 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+-- add string_interning ledger-end tracking column to parameters
+ALTER TABLE parameters
+  ADD COLUMN ledger_end_string_interning_id INTEGER;
+
+-- create temporary sequence to populate the string_interning table as migrating data
+CREATE SEQUENCE string_interning_seq_temp;
+-- add temporary index to aid string lookups as migrating data
+CREATE UNIQUE INDEX string_interning_external_string_temp_idx ON string_interning USING btree (external_string);
+
+-- add temporary migration function
```

Comment:
this data migration is verified locally by ensuring it generates the same string_interning table as it was populated by the write side

for reviewers: this needs to confirm to logic in DbDtoToStringsForInterning.